### DISCUSSION TITLE
🙏 Help: KoliBri already registered?

### DISCUSSION THREAD

**Discussion by deleonio** [public-ui/kolibri#6543] [Q&A] [answered]
### Summary

Is there a way to check from the outside whether KoliBri has already been registered? We currently have an ugly workaround for this, but surely there is a better way. Do you have a helper for this?

```
function isKoliBriRegistered() {
  return document.createElement(kol('button')).constructor !== HTMLElement
}
```

**Comment by Chrisdo82** [comment]
@deleonio  gibt es hierzu schon einen neueren Stand von euch?

**Reply by deleonio** [reply]
Hi @Chrisdo82,

ehrlich gesagt nein. Ich nehme es mal zum nächsten Call mit. Bitte frage mich in unserem nächsten Austausch noch einmal.

Viele Grüße
Martin

**Reply by deleonio** [reply, marked as answer]
Hi @Chrisdo82,

das Feature ist doch schon da. Kannst Du es ggf. malfür Euren Case ausprobieren?

Siehe https://github.com/public-ui/kolibri/blob/c4fbb5c5d39afe5045ee6f372a143646235bbde8/packages/components/src/index.ts#L4

**Reply by Chrisdo82** [reply]
Ja das passt, danke @deleonio
